rotkehlchen/tasks/manager.py

```diff
@@ -552,8 +552,9 @@ def _maybe_query_produced_blocks(self) -> Optional[list[gevent.Greenlet]]:
         """Schedules the blocks production query if enough time has passed"""
         if (
             self.chains_aggregator.get_module('eth2') is None or
-            self.chains_aggregator.beaconchain.produced_blocks_lock.locked() or
-            len(indices := self.chains_aggregator.beaconchain.get_outdated_validators_to_query_for_blocks()) == 0  # noqa: E501
+            (beeaconchain := self.chains_aggregator.beaconchain).is_rate_limited() or
```

Comment:
why add another variable here? Just to make the line smaller? Don't do that. Also it's `beaconchain`. Not `beeaconchain`